cmd/proxy/actions/app.go

```diff
@@ -109,8 +109,6 @@ func App() (*buffalo.App, error) {
 		}
 		app.Use(T.Middleware())
 
-		app.GET("/", homeHandler)
```

Comment:
homeHandler  func will need to be removed as well

Reply:
Cool, I was waiting for confirmation :smile: when we add tracing we can add a span to the `proxyHomeHandler` if desired.